lib/features/stream/components/chat/index.tsx

```diff
@@ -68,67 +29,33 @@ export function Chat() {
         }
     }, [messages]);
 
-    //send message when press enter
-    const handleKeyDown = (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
-        if (e.key === "Enter" && !e.shiftKey) {
-            e.preventDefault();
-            handleSendMessage();
-        }
-    };
-
-    //expand textarea
-    useEffect(() => {
-        if (textareaRef.current) {
-            textareaRef.current.style.height = "inherit";
-            textareaRef.current.style.height = `${textareaRef.current.scrollHeight}px`;
-        }
-    }, [newMessage]);
-
     return (
-        <div className="flex h-[calc(100vh-5rem)] w-[500px] flex-col justify-between rounded-xl border border-gray-700 bg-transparent text-white">
-            <div className="flex justify-between border-b border-gray-700 p-2">
-                <TooltipModel content="Collapse" side="bottom">
-                    <button onClick={onChangeChatComponent}>
-                        <ArrowRightToLine />
-                    </button>
-                </TooltipModel>
-                <p className="text-lg font-semibold">Live chat</p>
-                <Settings />
-            </div>
+        <div className="flex w-full flex-col justify-between rounded-xl border border-gray-700 bg-transparent text-white">
+            <CustomChannelHeader />
             <ScrollArea
                 ref={scrollAreaRef}
                 className="h-[calc(100vh-12rem)] px-4"
             >
-                <div className="flex flex-col space-y-2 py-2">
-                    {messages.map((msg, index) => (
-                        <ChatMessage
-                            key={index}
-                            avatar="/user.svg"
-                            message={msg.message}
-                            userName={msg.username}
-                        />
-                    ))}
-                </div>
+                {messages && messages.length > 0 ? (
+                    <MessageList
+                        showUnreadNotificationAlways={false}
+                        disableDateSeparator={false}
+                        Message={ChatMessage}
+                    />
+                ) : (
+                    <></>
+                )}
+                {messages && messages.length > 0 ? (
+                    <MessageList
+                        showUnreadNotificationAlways={false}
```

Comment:
Bị lặp nè. Chỗ này ko cần dùng toán tử 3 ngôi, messages && messages.length > 0 && (
                    <MessageList
                        showUnreadNotificationAlways={false}
                        disableDateSeparator={false}
                        Message={ChatMessage}
                    />
                )